api/current.txt

```diff
@@ -23,6 +23,7 @@ package androidx.content {
 
   public final class ContextKt {
     ctor public ContextKt();
+    method public static java.util.List<java.lang.String> getPermissions(android.content.Context);
```

Comment:
Why this change?

Reply:
sorry, it's a mistake, I have already removed it